### ISSUE TITLE
I found a Issue with Dimensional wrench  and Dimensional Pocket  Block.

### ISSUE THREAD

**Issue by Beyondall01** [TheRealZeher/DimensionalPockets-II#20]
I recorded what I found and made a video to show you whats happening. I'll send you a link to the video (youtube video unlisted) to your curseforge in private messages. To explain when you place the Dimensional Pocket Block normally this creates a pocket in the pocket dimension. If you use  the Dimensional Wrench to make the Dimensional Pocket Block a item and place it back where it was first place the block then everything remains normal just one pocket in the pocket dim however if you place it some where else that one Dimensional Pocket Block then create's a new Pocket in the pocket dim instead of you returning to the first pocket that was create.  Whatever you put on that first or 2 etc.. pocket in the pocket dim if you don't places the dimensional pocket block in the exact places you place it the first time, Your not gonna be able to go back to your pocket that has your stuff at less you go to gamemode 3 (spectator mode) to the go to  inside that pocket you want to get in to and the put your self in gamemode 0 to get your stuff but even then if you try to get out the pocket when you shift left click it said's you're trapped someone broke  your pocket. See the video so and yes the pocket dimension still appears in the over world in large scale in some of my other testings that I have done. _### **Hope this helps.**_ P.s I found a other bug with Dimensional doors mod: https://minecraft.curseforge.com/projects/dimensionaldoors I'll send the code with the crash soon.

**Comment by TheRealZeher** [comment]
Hi, I'm am currently aware of the problem you have described. I have been without a working computer for some time now, however I've just bought, myself a new laptop and are transferring the code now. This issue already has an open issue so I will close the issue, and make a new one for the dimensional doors mod crash when you provide me the crash report. Thanks for letting me know. TheRealZeher.

**Comment by Beyondall01** [comment]
> Hi, I'm am currently aware of the problem you have described. I have been without a working computer for some time now, however I've just bought, myself a new laptop and are transferring the code now. This issue already has an open issue so I will close the issue, and make a new one for the dimensional doors mod crash when you provide me the crash report. Thanks for letting me know. TheRealZeher.



Ok thank you I'll send you the code of the crash for the dimensional Door mod. I will now not send you the video now because you already know so thank you for your hard work and time.

**Comment by Beyondall01** [comment]
TheRealZeher. I did some testing with Dimensional Door mod and Dimensional Pocket II mod and no Crash so I added the simple void mod and test it your mod with it by entering the pocket dim then the door pocket and then simple void world mod and I have not gotten a crash. I must gotten a crash because of having to much mods with a lot of data or Big mods but useful mods like NEI mod and when I enter the pocket it must of overloaded or something that made it crash, but im using JEI now with Dimensional pocket II, the Dimensional Door mod and the simple void world mod and everything is running fine. So the Dimensional Pocket II and Dimensional Door are fine together.
